deploy/database/updates/00344_custom_recipe_01.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO buttonset (id, name, sort_order)
+VALUES (20000, 'CustomBM', 20);
+
+INSERT INTO button (id, name, recipe, btn_special, tourn_legal, set_id, flavor_text, sort_order)
+VALUES (11000, 'CustomBM', '', 0, 0, 20000, 'This button has a custom recipe', 0);
```

Comment:
There's a problem here --- this flavor text is defined in the update version, but not in `data.button.sql`.  Please fix so these match (and feel free to rebase right away, since it's a one-liner).